scripts/llnl/common_build_functions.py

```diff
@@ -593,6 +596,39 @@ def build_devtools(builds_dir, timestamp, short_path, report_to_stdout = False):
     return res
 
 
+def ensure_on_lc_and_group_permissions():
```

Comment:
i think we should still allow the use of these scripts on non-LC machines. for example, this is breaking the github actions CI. as long as SYS_TYPE is set, these scripts work completely fine.